examples/package.json

```diff
@@ -0,0 +1,17 @@
+{
+  "name": "acuvity-ts-examples",
+  "version": "1.0.0",
+  "main": "basic_text.js",
+  "type": "module",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
```

Comment:
It's customary in the JavaScript/TypeScript world to have run targets here which would execute the examples. A small README either here in the examples folder (which personally I would prefer) or in the main README would make sense.

Reply:
About run targets let me read about it. 

yup, let me add a readme in examples. Its not that straight forward as python, I had do to some links also to make it work, I should document that.